metro.config.js

```diff
@@ -10,8 +10,40 @@ module.exports = {
     getTransformOptions: async () => ({
       transform: {
         experimentalImportSupport: false,
-        inlineRequires: true,
-      },
-    }),
+        inlineRequires: true
+      }
+    })
   },
+  resolver: {
+    assetExts: [
```

Comment:
@Woynert Is it possible to only define the extensions we'll need? For example, I don't think we'll need the html or pdf extensions.